src/main/java/seedu/programmer/model/student/Lab.java

```diff
@@ -61,8 +61,8 @@ public LabTotal getLabTotal() {
         return totalScore;
     }
 
-    public String getLabNumValue() {
-        return labNum.getLabNum().toString();
+    public int getLabNumValue() {
+        return labNum.getLabNum();
```

Comment:
I think this method is intended to return the string representation for labNum, which is specifically used in `JsonAdaptedLab` for storage purpose.  Maybe it is better to create a separate method with a diff name?

Reply:
Yup I was thinking of creating a separate method, but not sure if it's too much of an overkill since the String representation is only used in `JsonAdaptedLab`? 

I think the disadvantage is that it might confuse other programmers if there are too many different variations they can get. For example, we already have a `getLabNum` method, so I thought the `getLabNumValue` should return the int value, but it returned a String instead. To address this, I could create a `getLabNumString` method, would that be better?

Reply:
I think the disadvantage is that it might confuse other programmers if there are too many different variations they can get. For example, we already have a `getLabNum` method, so I thought the `getLabNumValue` should return the int value, but it returned a String instead. To address this, I could create a `getLabNumString` method, would that be better?

Reply:
Yep I agree, either create `getLabNumString` or using String.valueOf(..) for labNum, actualScore and totalScore in `JsonAdaptedLab` is fine I think, since the string representation was only used once

Reply:
> I think the disadvantage is that it might confuse other programmers if there are too many different variations they can get. For example, we already have a `getLabNum` method, so I thought the `getLabNumValue` should return the int value, but it returned a String instead. To address this, I could create a `getLabNumString` method, would that be better?

I followed the naming convention in `JsonAdaptedStudent` when I wrote these methods, now I realize it is indeed confusing and thanks for pointing that out!

Reply:
How about we don't use Strings in JsonAdaptedLab? Since the data aren't strings anyway, we can simply store them as numbers in the json file

Reply:
> How about we don't use Strings in JsonAdaptedLab? Since the data aren't strings anyway, we can simply store them as numbers in the json file

I am not sure if this is feasible but it is definitely ideal

Reply:
> How about we don't use Strings in JsonAdaptedLab? Since the data aren't strings anyway, we can simply store them as numbers in the json file

Yep that would be ideal, thanks!